datahub-web-react/src/graphql/chart.graphql

```diff
@@ -96,3 +96,102 @@ query getChart($urn: String!) {
         }
     }
 }
+
+mutation updateChart($input: ChartUpdateInput!) {
+    updateChart(input: $input) {
```

Comment:
Eventually we can fragmentize the projections I'd imagine

Reply:
I was having some trouble with fragments and apollo mocks- need to look into this more (or get off apollo mocks)